libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/internal/MapboxTripNotification.kt

```diff
@@ -217,6 +212,7 @@ class MapboxTripNotification constructor(
      * @param applicationContext the application's [Context]
      * @return [PendingIntent] to opening application
      */
+    @SuppressLint("UnspecifiedImmutableFlag")
```

Comment:
At the moment we have the code as below:
```
PendingIntent.getBroadcast( applicationContext, 0, endNavigationBtn, 0)
```

The recommendation as per the above link is to either use `FLAG_MUTABLE` or `FLAG_IMMUTABLE` instead of 0. But these flags are available starting Android M. Hence, I cannot use these flags and would have to stick to `0`. Using `0` throws a warning. Hence the suppress warning.

Reply:
Ohh, that makes sense. Thanks for clarifying.